scan

```diff
@@ -290,11 +321,28 @@ def perform_scan(params: Tuple[Any, str, dict, dict, dict]):
 
             # Capture local start and end times around scan.
             meta['start_time'] = scan_utils.local_now()
+
+            # Drop the fastcache from the scan_environment before
+            # (potentially) sending to Lambda, since it may be huge
+            scan_environment.pop(FAST_CACHE_KEY, None)
+
             data = scan_method(scanner, domain, handles, scan_environment, options, meta)
+
             meta['end_time'] = scan_utils.local_now()
             meta['duration'] = meta['end_time'] - meta['start_time']
 
         if data:
+            if hasattr(scanner, 'fast_cache_support'):
+                # Update the cache
+                if FAST_CACHE_KEY not in environment:
+                    environment[FAST_CACHE_KEY] = []
+                # I hate having to use isinstance here, but I don't
+                # see another way
```

Comment:
Didn't we standardize the return value as a list (of lists) here? (It can also return `None`.)

Reply:
I don't think that's true.  Neither `scanners/pshtt.py` nor `scanners/trustymail.py`return a `list`.

Reply:
Ah, I was thinking of `to_rows` here.